classes/class-uninstall.php

```diff
@@ -218,7 +220,7 @@ private function delete_blog_user_meta( $blog_id = 1 ) {
 		// Specific user meta.
 		foreach ( $this->user_meta as $meta_key ) {
 			$wpdb->query(
-				$wpdb->prepare( "DELETE FROM {$wpdb->usermeta} WHERE meta_key = {$wpdb->prefix}%s;", $meta_key )
+				$wpdb->prepare( "DELETE FROM {$wpdb->usermeta} WHERE meta_key = %s;", $wpdb->prefix . $meta_key )
```

Comment:
This is a bug I detected while adding tests to multisite uninstallation. That probably never worked as intended, it generates a meta key like `stream'my_meta-key'` (prefix outside of quotes).